src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -115,6 +118,23 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     }
   }
 
+  private def assertCallFqnWellFormed(callFqn: FullyQualifiedName): Future[(String, String)] = {
```

Comment:
We should have a comment describing what the body of the tuple is. A type would be nice but even I don't think it's necessary here